barretenberg/cpp/src/barretenberg/polynomials/pow.hpp

```diff
@@ -158,4 +117,68 @@ template <typename FF> struct PowPolynomial {
         });
     }
 };
+/**<
+ * @struct PowPolynomial
+ * @brief Implementation of the methods for the \f$pow_{\ell}\f$-polynomials used in ProtoGalaxy and
+\f$pow_{\beta}\f$-polynomials used in Sumcheck.
```

Comment:
They are different, although they agree on some arguments which is very useful. But say, it is not clear what it means to evaluate pow_i(X_0,\ldots,X_{d-1}) [if we only keep PG pow] at the challenge point (u_0,\ldots,u_{d-1}), because it's the index i that gets replaced by the challenge in Sumcheck. Does it make sense to you?